ui/components/app/srp-quiz-popover/SRPQuiz/SRPQuiz.tsx

```diff
@@ -0,0 +1,268 @@
+/* eslint-disable @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports, import/no-commonjs */
+import React, { useCallback, useContext, useMemo, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import {
+  MetaMetricsEventCategory,
+  MetaMetricsEventKeyType,
+  MetaMetricsEventName,
+} from '../../../../../shared/constants/metametrics';
+import { MetaMetricsContext } from '../../../../contexts/metametrics';
+import {
+  BlockSize,
+  IconColor,
+  TextAlign,
+} from '../../../../helpers/constants/design-system';
+import { REVEAL_SEED_ROUTE } from '../../../../helpers/constants/routes';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import {
+  BUTTON_VARIANT,
+  Icon,
+  IconName,
+  IconSize,
+  Modal,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+} from '../../../component-library';
+import QuizContent from '../QuizContent';
+import { JSXDict, QuizStage } from '../types';
+
+const wrongAnswerIcon = (
+  <Icon
+    size={IconSize.Xl}
+    name={IconName.Warning}
+    color={IconColor.errorDefault}
+    textAlign={TextAlign.Center}
+    width={BlockSize.OneTwelfth}
+  />
+);
+
+const rightAnswerIcon = (
+  <Icon
+    size={IconSize.Xl}
+    name={IconName.Confirmation}
+    color={IconColor.successDefault}
+    textAlign={TextAlign.Center}
+    width={BlockSize.OneTwelfth}
+  />
+);
+
+const openSupportArticle = (): void => {
+  global.platform.openTab({
+    url: 'https://support.metamask.io/hc/en-us/articles/4404722782107',
```

Comment:
We should probably add this URL to `ui/helpers/constants/zendesk-url.js` and reference it here.